BE/GovernmentEntities/NorthAmericanJurisdiction/MXGovernmentEntitiesAndJurisdictions.rdf

```diff
@@ -63,20 +63,21 @@
 		<owl:imports rdf:resource="https://www.omg.org/spec/LCC/Countries/CountryRepresentation/"/>
 		<owl:imports rdf:resource="https://www.omg.org/spec/LCC/Countries/ISO3166-1-CountryCodes/"/>
 		<owl:imports rdf:resource="https://www.omg.org/spec/LCC/Countries/Regions/ISO3166-2-SubdivisionCodes-MX/"/>
-		<owl:versionIRI rdf:resource="https://spec.edmcouncil.org/fibo/ontology/BE/20200701/GovernmentEntities/NorthAmericanJurisdiction/MXGovernmentEntitiesAndJurisdictions/"/>
+		<owl:versionIRI rdf:resource="https://spec.edmcouncil.org/fibo/ontology/BE/20210201/GovernmentEntities/NorthAmericanJurisdiction/MXGovernmentEntitiesAndJurisdictions/"/>
+		<skos:changeNote>The http://www.omg.org/spec/EDMC-FIBO/BE/20200701/GovernmentEntities/NorthAmericanJurisdiction/MXGovernmentEntitiesAndJurisdictions.rdf version of this ontology was modified to replace &apos;hasPartialSovereigntyOver&apos; with &apos;hasSharedSovereigntyOver&apos;.</skos:changeNote>
 		<fibo-fnd-utl-av:hasMaturityLevel rdf:resource="&fibo-fnd-utl-av;Release"/>
 	</owl:Ontology>
 	
 	<owl:NamedIndividual rdf:about="&fibo-be-ge-mxj;FederalGovernmentOfMexico">
 		<rdf:type rdf:resource="&fibo-be-ge-ge;FederalGovernment"/>
-		<rdfs:label>Federal Government of Mexico</rdfs:label>
+		<rdfs:label xml:lang="en">Federal Government of Mexico</rdfs:label>
+		<rdfs:label xml:lang="es-MX">Gobierno Federal de México</rdfs:label>
 		<rdfs:isDefinedBy rdf:resource="https://spec.edmcouncil.org/fibo/ontology/BE/GovernmentEntities/NorthAmericanJurisdiction/MXGovernmentEntitiesAndJurisdictions/"/>
 		<rdfs:seeAlso rdf:resource="http://www.gob.mx/"/>
-		<skos:definition>the Gobierno de la República, a federal democracy including three branches: executive, legislative, and judicial, which functions per the Constitution of the United Mexican States</skos:definition>
+		<skos:definition>federal presidential constitutional republic with shared sovereignty over the republic with the governments of the 31 individual Mexican states, which functions per the Constitution of the United Mexican States</skos:definition>
```

Comment:
Seems unnecessary to include the number of states in the definition. If a state were to be created or merged the Government would not change.